setup.py

```diff
@@ -2,7 +2,8 @@
 
 setup(
     name='pyfiresql',
-    version='0.3.9',
+    # version='0.3.9',
+    version='0.3.749',
```

Comment:
Honestly, I did not mean to commit this change.  I did it to confirm that I was working with the version expected locally.  This change can be safely removed.  But maybe want to bump the version number, at least?

Reply:
Thanks for all of your improvement efforts! If this improvement makes your project work, I don't mind accepting all of your changes. Your comments demonstrate a clear understanding of the code base; I trusted your changes.

Reply:
I am currently writing a suite of unit tests to confirm all of these changes are good from the client's standpoint.  So far so good, though.  I will keep you posted with results.